renovate.json

```diff
@@ -1,5 +1,12 @@
 {
   "extends": [
     "config:base"
+  ],
+
+  "packageRules": [
+    {
+      "matchPackagePatterns": ["^github.com/prometheus"],
```

Comment:
only Prometheus/prometheus and Prometheus/alertmanager should be impacted. It would be great to continue updating common and client_golang.

Reply:
Thanks! 

I've got so many projects than depend on Prometheus and finding a way to keep them up to date is a pain.  Have you got any tips? Has this been discussed anywhere?

Reply:
Yes, https://github.com/prometheus/prometheus/issues/8852